platforms-whitepaper/v1alpha1/60-platform-capabilities.md

```diff
@@ -1,3 +1,95 @@
+    <td>Documentation and code templates. APIs, Web UIs and CLIs. All the self-service experiences and  capabilities provided by a platform.</td>
+    <td>Backstage (UI), Kubernetes (API, CLI), Kratix (API), Crossplane (API), Operator Framework, Dapr, Cartographer</td>
+  </tr>
+  <tr>
+    <td>Automation for building and testing services and products (#4)</td>
+    <td>Automate build and test of digital products and services.</td>
+    <td>Tekton, Jenkins, Buildpacks, ko, buildah, Carvel, pact, testcontainers, testkube, kuttl</td>
+  </tr>
+  <tr>
+    <td>Automation for deploying and delivering services and products (#5)</td>
+    <td>Automate and control delivery of services and applications. For example, canary, blue/green, feature flags, post-deployment verification.</td>
+    <td>ArgoCD, Flux, keptn, Flagger, OpenFeature, kapp (Carvel), Konveyor</td>
+  </tr>
+  <tr>
+    <td>Observe products and services (#6)</td>
+    <td>Enable all infrastructure capabilities and product workloads to emit telemetry.  Gather, analyze and publish info to product stakeholders.</td>
+    <td>OpenTelemetry, Jaeger, Prometheus, Thanos, Fluentd, Grafana</td>
+  </tr>
+  <tr>
+    <td>Infrastructure services (#7)</td>
+    <td>Run application code. Connect application components. Persist data for applications</td>
+    <td>Kubernetes (containers), kubevirt (VMs), Knative (functions), Istio, Envoy, Contour, Cilium, Rook, Longhorn, OpenEBS</td>
+  </tr>
+  <tr>
+    <td>Data services (#8)</td>
+    <td>Persist structured data for applications</td>
+    <td>TiKV, Vitess, etcd, SchemaHero</td>
+  </tr>
+  <tr>
+    <td>Messaging services (#9)</td>
+    <td>Enable applications to communicate with each other asynchronously</td>
+    <td>CloudEvents, NATS, gRPC, Knative (events)</td>
+  </tr>
+  <tr>
+    <td>Identity and secret services (#10)</td>
+    <td>Ensure workloads have the locators and secrets to access and use provisioned resources and services. Enable services to identify themselves securely to other services</td>
+    <td>Directory, OAuth/OIDC, Hashicorp Vault, Dex, Keycloak, Pinniped, SPIFFE/SPIRE, servicebinding.io, cert-manager, Carvel secretgen</td>
+  </tr>
+  <tr>
+    <td>Security services (#11)</td>
+    <td>Observe runtime behavior and report/remediate anomalies. Verify builds and artifacts don't contain vulnerabilities. Constrain activities on the platform per enterprise requirements; notify and/or remediate aberrations</td>
+    <td>Falco, KubeArmor, StackRox, OPA, Kyverno, cloud custodian</td>
+  </tr>
+  <tr>
+    <td>Artifact storage (#12)</td>
+    <td>Store, publish and secure built artifacts for use in production. Cache and analyze third-party artifacts.  Store source code.</td>
+    <td>ArtifactHub, sigstore, Harbor, Dragonfly, OCI</td>
+  </tr>
+  <tr>
+    <td>Extensibility</td>
+    <td>Add APIs, controllers and portal pages to the platform to provide new functionality using the same patterns.</td>
+    <td>Operator Framework</td>
```

Comment:
In order to have at least a couple of examples in this category, I suggest adding [MetaController](https://github.com/metacontroller/metacontroller).